terminus/geometry/bounding_box.py

```diff
@@ -0,0 +1,33 @@
+from geometry.point import Point
+
+
+class BoundingBox(object):
+
+    def _normalize(self):
+        new_origin = Point(min(self.origin.x, self.corner.x),
```

Comment:
You want the max between the two points giving points an order, or a point like the one I wrote? (Yes, it's not the same, it depends on which opposite corners we give)